chapters/data/main.tex

```diff
@@ -72,293 +85,290 @@ \subsection{Data structure}\label{subsec:structure}
-well-known relationships that can be justified anecdotally: the longer a patient
+total costs (COST) and net costs (NetCost) with a value of 0.94. This high value
+indicates almost total positive linear correlation between these two variables,
+which makes sense given that the net cost of a spell is the total cost corrected
+for any reimbursable costs such as critical care costs (CRIT) and non-contracted
+income (NCI). Reimbursable costs are given as negative values in the dataset ---
+hence their distinctly negative correlation coefficients with the other
+variables. Typically, these deductible costs are small
+(see Table~\ref{tab:summary}) so a strong correlation between costs and net
+costs is to be expected.
+
+Other examples of strong correlation are those between length of stay
+(TRUE\_LOS) and ward and overhead costs (WARD and OVH respectively). These are
+well-known relationships that can be justified succinctly: the longer a patient
 spends in hospital, the more time they are likely to spend on a ward. Thus,
-incurring associated overheads like administrative chapter, cleaning costs and a
-larger proportion of rental costs. It should also be clear that these attributes
-all share a strong linear correlation with the net cost of a spell. This
-suggests that these costs and the length of stay are strong indicators of the
-net cost of treating someone, and may suggest that the remaining cost components 
+incurring associated overheads like administrative work, cleaning costs and a
+larger proportion of rental costs. It should also be clear that these three
+attributes all share a strong linear correlation with the net cost of a spell,
+suggesting that these costs and the length of stay are strong indicators of the
+net cost of treating someone, and may suggest that the remaining cost components
 make up a substantially smaller part of the net cost.
 
 
 \subsection{Measuring variation and importance in our cost components}
 
-The larger purpose of this chapter is to better understand the factors leading to
-some variation in the cost of treating patients so it would be fitting to
-investigate how this variation appears in each of the cost components. By doing
-so, a high-level indication of which departments and procedures that impose more
-(or less) variation can be established. Once a level of variation has been
-determined, the relative importance of that component and its variation can be
-assessed by considering the overall contribution that component makes to net
+The broader purpose of this chapter is to better understand the factors leading
+to variation in the cost of treating patients. Therefore, it would be fitting to
+investigate how this variation can be attributed to each of the cost components.
+By doing so, a high-level indication of which departments and procedures that
+impose more (or less) variation can be identified. Once a level of variation has
```

Comment:
Do they impose variation or do they "create variation"?

Reply:
Yeah, that's a good point. I think when I originally wrote this I was thinking about this from the perspective of a patient spell where it could appear that variation is imposed upon the spell.

That feels pretty clumsy now that I think about it, and misses the point of the analysis as something to do with the system. I'll change that now.

```suggestion
create more (or less) variation can be identified. Once a level of variation has
```